Makefile

```diff
@@ -13,7 +13,7 @@ ORG := jenkins-x
 ORG_REPO := $(ORG)/$(NAME)
 RELEASE_ORG_REPO := $(ORG_REPO)
 ROOT_PACKAGE := github.com/$(ORG_REPO)
-GO_VERSION := 1.18.4
+GO_VERSION := 1.19.3
```

Comment:
Change to: https://github.com/jenkins-x/jx/blob/7987112f6727f6072aaac2d94eea787f77a99efb/Makefile#L18